src/prometheus-net.DotNetRuntime.Tests/StatsCollectors/IntegrationTests/ExceptionStatsCollectorTests.cs

```diff
@@ -0,0 +1,67 @@
+using NUnit.Framework;
+using Prometheus.DotNetRuntime.StatsCollectors;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Prometheus.DotNetRuntime.Tests.StatsCollectors.IntegrationTests
+{
+    [TestFixture]
+    internal class ExceptionStatsCollectorTests : StatsCollectorIntegrationTestBase<ExceptionStatsCollector>
+    {
+        protected override ExceptionStatsCollector CreateStatsCollector()
+        {
+            return new ExceptionStatsCollector();
+        }
+
+        [Test]
+        public void Will_measure_when_occurring_an_exception()
+        {
+            // arrange
+            int divider = 0;
+            string exceptionMessage = string.Empty;
+
+            // act
+            try
+            {
+                var result = 1 / divider;
+            }
+            catch (Exception ex)
+            {
+                exceptionMessage = ex.Message;
```

Comment:
So this is what I was warning about in a previous comment- we shouldn't be using the `Message` of an exception in labels as it's too verbose. Please change this so that the label value only includes the type of an exception (i.e. in this case, `System.DivideByZeroException`)